client/js/libs/handlebars/parse.js

```diff
@@ -114,7 +117,7 @@ module.exports = function parse(text, users) {
 			const escapedChannel = Handlebars.Utils.escapeExpression(textPart.channel);
 			return `<span class="inline-channel" role="button" tabindex="0" data-chan="${escapedChannel}">${fragments}</span>`;
 		} else if (textPart.emoji) {
-			return `<span class="emoji">${fragments}</span>`;
+			return `<span class="emoji" aria-label="${emojis[textPart.emoji]}">${fragments}</span>`;
```

Comment:
Also, I tried to add a tooltip for this, but because we `overflow: hidden` to avoid Zalgo, tooltips mostly don't show up, that's unfortunate. I don't know if/how this can be done.

In the meantime (unless you or something else has a solution), could you add `title="${...}"` at the very least?

Reply:
This more than likely will be labeling as undefined for ZWJ modifiers that we don`t have in the map.

Reply:
This reminds me... We should update the emoji for the new ones. They have been finalised but not in any yet, but probably worth updating before that. We can still send them.

Reply:
@xPaw, we can do `undefined` check, no big deal imo.

Reply:
What should the undefined check look like? I'm not sure how the parser handles ZWJ modifiers